### PR TITLE
LYNX-153: Documentation for attributesList query

### PR DESCRIPTION
**Pull request by loginesta**
## Purpose of this pull request

This pull request (PR) is to add the documentation for GraphQl attributesList query

This PR is related to this jira ticket: [LYNX-100](https://jira.corp.adobe.com/browse/LYNX-100)

## Affected pages

New page: graphql/schema/attributes/queries/attributes-list.md/

### CODE REVIEW

File: src/pages/graphql/schema/attributes/queries/attributes-list.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+For instance: `CUSTOMER`, `CUSTOMER_ADDRESS`, `CATALOG_CATEGORY`, `CATALOG_PRODUCT`, `INVOICE`, etc.
+
+## Output attributes
+
+The `AttributesMetadataOutput` object contains the following attributes:
+
+Attribute | Data Type | Description
+--- |---| ---
+`items` | [CustomAttributeMetadataInterface!]! | Requested attributes metadata.
```
Comment (keharper) [inline src/pages/graphql/schema/attributes/queries/attributes-list.md:105]:
```suggestion
`items` | [CustomAttributeMetadataInterface!]! | Requested attributes metadata
```

File: src/pages/graphql/schema/attributes/queries/attributes-list.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+`errors` | [AttributeMetadataError!]! | Errors of retrieving certain attributes metadata.
```
Comment (keharper) [inline src/pages/graphql/schema/attributes/queries/attributes-list.md:106]:
```suggestion
`errors` | [AttributeMetadataError!]! | Errors of retrieving certain attributes metadata
```

**Comment by keharper** [comment]
I just realized I could check the DB, and I found a promising list of values in the EAV_ENTITY_TYPE.ENTITY_TYPE_CODE column.  Is the value specified in `entityType` case sensitive?

I'd rather not point to the DB table, because it's not always available.

**Comment by loginesta** [comment]
@keharper you're right! the eav_attribute has a relation with EAV_ENTITY_TYPE table, matching the values on the ENTITY_TYPE_CODE column.

I have just double-checked that, because I was not 100% sure if we can retrieve all the entity type codes in that DB table.
Currently, we can retrieve the ones that are defined in any schema.graphqls as follows:
```
enum AttributeEntityTypeEnum {
    CATALOG_PRODUCT
}
```

According to this, they are just `CUSTOMER`, `CUSTOMER_ADDRESS` and `CATALOG_PRODUCT`.
So, to be on the safe side, I would just add these tree as possible values in the documentation pages.

File: src/pages/graphql/schema/attributes/queries/attributes-list.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+## Input attributes
+
+The `AttributeEntityTypeEnum` object contains the following attributes:
+
+Attribute | Data Type | Description
+--- |---| ---
+`entityType` | String | The type of entity that defines the attribute.
+
+The possible values for this attribute are populated by the modules introducing EAV entities.
```
[the remark this answers is not in the document]
Comment (loginesta) [reply src/pages/graphql/schema/attributes/queries/attributes-list.md:95]: Currently, we can retrieve `CUSTOMER`, `CUSTOMER_ADDRESS` and `CATALOG_PRODUCT`, which are the ones defined in a schema.graphqls as follows:
```
enum AttributeEntityTypeEnum {
    CUSTOMER
    CUSTOMER_ADDRESS
}
```

**Comment by keharper** [comment]
I didn't notice the value was an enum--I thought it was string.

**Review by keharper — APPROVED** [review]

**Pull request merged**